docs/getting-started/connect-server-to-github.md

```diff
@@ -1,70 +1,88 @@
-GITHUB_APP_APP_ID = [github-app-app-id]
-GITHUB_APP_PRIVATE_KEY = [github-app-private-key]
-GITHUB_APP_INSTALLATION_URL = [github-app-installation-url]
-```
+2. Enter the fields as follows:
+   - **Application name** - `[your-github-username]-amplication-local`
+   - **Homepage URL** - `http://localhost:3000`. _If you are hosting the Amplication client on any other address, use the specific address._
+   - **Application description** - optional
+   - **Authorization callback URL** - `http://localhost:3001`. _If you are hosting the Amplication server on any other address, use the specific address._
+3. Hit Save.
 
-4. Replace `[use-secret-manager]` with the **Client secret** of the new GitHub application.
-5. Replace `[github-app-client-id]` with the **Client ID** of the new GitHub application.
+![](./assets/app-settings.png)
 
-6. Restart Amplication server.
+4. Hit `Generate new client secret` and copy the resulting secret
+
+### Step 2: Configure Amplication server and client to work with the new GitHub app
+
+5. Clone [/packages/amplication-server/.env](https://github.com/amplication/amplication/blob/master/packages/amplication-server/.env) into `/packages/amplication-server/.env.local`
+6. Update `/packages/amplication-server/.env.local` with the following variables
+   ```sh
+   GITHUB_REDIRECT_URI=http://localhost:3000/github/callback
+   GITHUB_CLIENT_ID="replace with the github auth application client id"
+   GITHUB_CLIENT_SECRET="replace with secret created as step 4""
+   ```
+7. Clone [/packages/amplication-client/.env](https://github.com/amplication/amplication/blob/master/packages/amplication-client/.env) into `/packages/amplication-client/.env.local`
+8. Update `/packages/amplication-client/.env.local` with the following variables
+   ```sh
+   NX_REACT_APP_GITHUB_AUTH_ENABLED=true
+   NX_REACT_APP_GITHUB_CONTROLLER_LOGIN_URL=http://localhost:3000/github
+   ```
+9. Restart Amplication server and client.
+
+## GitHub app for sync
+
+Amplication leverage GitHub Apps to integrate with GitHub and sync your generated code to the git GitHub remotes.
+
+### Step 1: Create a new GitHub App
+
+1. Login in your GitHub account and create a new GitHub App [here](https://github.com/settings/apps/new)
+
+![](./assets/register-new-github-app.png)
+
+2. Enter the fields as follows:
+   - **Application name** - `[your-github-username]-amplication-local`
+   - **Homepage URL** - `http://localhost:3000`. _If you are hosting the Amplication client on any other address, use the specific address._
+   - **Application description** - optional
+   - **Callback URL** - `http://localhost:3001`. _If you are hosting the Amplication server on any other address, use the specific address._
+   - **Webhook > Active** - "unchecked"
+   - **Permissions > Repository permissions**
+     - **Administration** - `Access: Read and Write`
+     - **Checks** - `Access: Read and Write`
+     - **Content** - `Access: Read and Write`
+3. Hit Save
```

Comment:
To remain consistent with the style throughout the docs, please do not use **Hit**. 

Instead, use  **Click** or **Select**

Reply:
addressed.